src/components/Borrow/SwitchBorrow/style.ts

```diff
@@ -0,0 +1,294 @@
+    maxHeight: '34px'
+  },
+  buttonBorrow: {
+    ...typography.subtitle1,
+    color: colors.white.main,
+    borderRadius: '5px',
+    minWidth: '95px'
+  },
+  repayButton: {
+    ...typography.subtitle1,
+    color: colors.navy.background,
+    borderRadius: '5px',
+    minWidth: '95px'
+  },
+  noActiveButton: {
+    colors: colors.navy.info,
+    background: 'transparent'
+  },
+  middleGrid: {
+    display: 'flex',
+    flexDirection: 'row',
+    padding: '24px 24px',
+    [theme.breakpoints.down('md')]: {
+      flexDirection: 'column',
+      padding: '24px 16px'
+    }
+  },
+  collateralContainer: {
+    display: 'flex',
+    width: '125%',
+    [theme.breakpoints.down('md')]: {
+      paddingBottom: 28,
+      width: '100%'
+    },
+    [theme.breakpoints.down('xs')]: {
+      paddingBottom: 20
+    }
+  },
+  title: {
+    ...typography.subtitle2,
+    color: colors.navy.grey,
+    paddingBottom: 16,
+    [theme.breakpoints.down('xs')]: {
+      ...typography.caption1,
+      paddingBottom: 8
+    }
+  },
+  desc: {
+    ...typography.subtitle2,
+    color: colors.navy.info,
+    paddingTop: 12,
+    [theme.breakpoints.down('xs')]: {
+      ...typography.caption1,
+      paddingTop: 8
+    }
+  },
+  divider: {
+    background: colors.navy.darkGrey
+  },
+  cRatioGrid: {
+    display: 'flex',
+    flexDirection: 'column',
+    minWidth: 'max-content',
+    padding: '0 16px',
+    justifyContent: 'center',
+    [theme.breakpoints.down('md')]: {
+      padding: '0 0 0 16px'
+    },
+    [theme.breakpoints.down('xs')]: {
+      padding: '0 0 0 4px'
+    }
+  },
+  cRatioBack: {
+    maxHeight: '64px',
+    marginTop: '3px',
+    display: 'flex',
+    flexDirection: 'column',
+
+    [theme.breakpoints.down('xs')]: {
+      maxHeight: '48px',
+      marginTop: '1px'
+    }
+  },
+  cRatioButton: {
+    ...typography.body1,
+    background: colors.navy.dark,
+    borderRadius: '0 0 8px 8px',
+    width: '100%',
+    padding: '4px 6px',
+    [theme.breakpoints.down('xs')]: {
+      ...typography.body3
+    }
+  },
+  endIcon: {
+    color: colors.white.main,
+    width: '24px',
+    margin: '0 0 0 2px',
+    [theme.breakpoints.down('xs')]: {
+      width: '13px'
+    }
+  },
+  cRatioTitle: {
+    ...typography.body4,
+    color: colors.navy.grey,
+    background: colors.navy.dark,
+    padding: ' 4px 21px 5px 21px',
+    borderRadius: '8px 8px 0 0',
+    display: 'flex',
+    justifyContent: 'center',
+    [theme.breakpoints.down('xs')]: {
+      ...typography.caption1,
+      padding: ' 4px 10px 5px 10px'
+    }
+  },
+  input: {
+    minHeight: '64px',
+
+    '& $button:first-child': {
+      [theme.breakpoints.up('md')]: {
+        minWidth: '107px',
+        borderRadius: '6px'
+      }
+    },
+    [theme.breakpoints.down('xs')]: {
+      minHeight: '48px'
+    }
+  },
+  popover: {
+    maxWidth: '90px',
+    borderRadius: '5px'
+  },
+  popoverBack: {
+    background: colors.navy.dark,
+    border: '1px solid #292956',
```

Comment:
use colors from theme, check this everywhere

Reply:
I changed where I could, some colors do not correspond to the theme